hawkular/metrics.py

```diff
@@ -124,7 +127,13 @@ def _clean_metric_id(metric_id):
         return quote(metric_id, '')
 
     def _get_base_url(self):
-        return "{0}://{1}:{2}/{3}/".format(self.scheme, self.host, str(self.port), self.path)
+        scheme = self.scheme.lower()
```

Comment:
@simon3z
I is :-) , I will try on my centos, and if it's just on my fedora i will remove it.

p.s. if @jpkrohling will show me another way to do this, i will close this PR.